tests/services/test_auth.py

```diff
@@ -1,7 +1,8 @@
 from jose import jwt
 from jose.constants import ALGORITHMS
 
-from contxt.auth.jwt import AuthTokenValidator, ContxtAuthTokenValidator, AuthError
+from contxt.auth.jwt import (AuthError, AuthTokenValidator,
+                             ContxtAuthTokenValidator)
 
```

Comment:
I know time is scarce... but would be awesome to have unit tests for the automatic token renewal

Reply:
agreed, i'm working on that next